3185 unify pipeline tag

pwilczynskiclearcode opened the pull request:
**What does this pull request do? Explain your changes.**
Unifies the "pipeline" prometheus metric tag produced by `AIRequestError`

Fixes #3185


**Checklist:**
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] Read the [contribution guide](./CONTRIBUTING.md)
- [ ] `make` runs successfully
- [ ] All tests in `./test.sh` pass
- [ ] README and other documentation updated
- [ ] [Pending changelog](./CHANGELOG_PENDING.md) updated

pwilczynskiclearcode committed:
[3185] Unify pipeline tag on prometheus metrics

leszko commented on monitor/census.go:
```diff
@@ -1924,3 +1927,7 @@ func FastVerificationFailed(ctx context.Context, uri string, errtype int) {
 	}
 }
+
+func normilizePipelineTag(pipeline string) string {
```
Typo
```suggestion
func normalizePipelineTag(pipeline string) string {
```

leszko reviewed (approved):
LGTM.

@rickstaa should we just merge it or should we notify anyone that the label is changed?

rickstaa committed:
feat(ai): unify AI remote worker pipeline tag

This commit ensures that the new metrics functions also use the
`normalizePipelineTag` method to ensure consistent pipeline naming in
the metrics.

rickstaa reviewed (approved):
@pwilczynskiclearcode Great catch! I've implemented your suggested change to the new AI remote worker metrics functions. One additional point to consider is whether we want to ensure consistent behavior in the [ai_process.go](https://github.com/livepeer/go-livepeer/blob/16de762f9376f85f115cef377c70502f8cdc8b4f/server/ai_process.go#L1119) file. Currently, we're inserting a capability string instead of a pipeline string, as we are in [census.go](https://github.com/livepeer/go-livepeer/blob/16de762f9376f85f115cef377c70502f8cdc8b4f/monitor/census.go#L1889). 

I'm okay with handling this discrepancy in the census package for now, but if we think it's better to also address this in the core codebase, we can include a change in a subsequent pull request. @leszko, @pwilczynskiclearcode — let me know what you think!

rickstaa commented:
@pwilczynskiclearcode, @leszko we can also change `pipeline string` to `pipelineTag string` to make the behavoir more clear from outside the census package.

leszko committed:
Minor change

leszko committed:
Change census.go to not use cap, but always pipeline

leszko commented:
Added one commit https://github.com/livepeer/go-livepeer/pull/3191/commits/421aa2cb6eb5e14fd5d5ce1856bca6668963a79c

I think I like the suggestion from @rickstaa to have this conversion outside the census code. We should use pipeline naming consistently through the code.

@pwilczynskiclearcode If you like my change, feel free to merge (after all CI builds pass).

leszko committed:
Merge branch 'ai-video' into 3185-unify-pipeline-tag

leszko committed:
Merge branch 'ai-video' into 3185-unify-pipeline-tag
